src/approaches/gnn_approach.py

```diff
@@ -225,7 +225,8 @@ def load(self, online_learning_cycle: Optional[int]) -> None:
         self._load_output_specific_fields_from_save_info(info)
 
     def _predict(self, state: State, atoms: Set[GroundAtom],
-                 goal: Set[GroundAtom]) -> _Output:
+                 goal: Set[GroundAtom], memory: Dict) -> _Output:
+        del memory  # unused
```

Comment:
is the idea that this will actually get used in the future?

Reply:
not by the GNN, but the memory will be used by the open loop LLM approach (to store the plan)

Reply:
this was confusing to me at first. but i see, you need to update this because gnn metacontroller approach will inherit it and also subclasses the nsrt metacontroller approach. that sucks

Reply:
yeah this may not have been the best design after all. one day maybe we will refactor to not use the metacontroller at all. but for now this works